CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Changed
 - Made `HiveRunnerScript` constructor public.
 - Made `scriptsUnderTest` variable in `HiveRunnerExtension` protected so it can be used in [MutantSwarm](https://github.com/HotelsDotCom/mutant-swarm).
+- Refactored `setScriptsUnderTest` method in `HiveShellBuilder`.
```

Comment:
```suggestion
```
You didn't change the behaviour of the method so this isn't something end users need to know about so you can remove it.